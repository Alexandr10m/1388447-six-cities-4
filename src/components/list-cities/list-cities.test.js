import React from "react";
import renderer from "react-test-renderer";
import ListCities from "./list-cities.jsx";
import {BrowserRouter} from "react-router-dom";


const props = {
  currentCity: `Amsterdam`,
  onCityClick: ()=>{},
};

describe(`Snapshot of ListCities`, () => {
  it(`ListCities should render correctly`, () => {
    const tree = renderer
      .create(
          <BrowserRouter>
            <ListCities
              {...props}
            />
          </BrowserRouter>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
