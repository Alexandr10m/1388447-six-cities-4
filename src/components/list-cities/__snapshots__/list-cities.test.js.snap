// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of ListCities ListCities should render correctly 1`] = `
<ul
  className="locations__list tabs__list"
  onClick={[Function]}
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="/Paris"
      onClick={[Function]}
    >
      <span>
        Paris
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="/Cologne"
      onClick={[Function]}
    >
      <span>
        Cologne
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="/Brussels"
      onClick={[Function]}
    >
      <span>
        Brussels
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="/Amsterdam"
      onClick={[Function]}
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="/Hamburg"
      onClick={[Function]}
    >
      <span>
        Hamburg
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="/Dusseldorf"
      onClick={[Function]}
    >
      <span>
        Dusseldorf
      </span>
    </a>
  </li>
</ul>
`;
